README.md

```diff
@@ -1,16 +1,123 @@
-# Title
+# Method Scope Lab
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+- Define a local variable with the expected value
+- Recognize the restrictions of local variable scope
+- Define a method that outputs a popular catch phrase
+- Define a method that takes in an argument
 
 ## Introduction
 
-## SWBAT 1
+We're going to build a catch phrase generator. Using knowledge of methods,
+scope, and variables, we're going to create a few methods that print a catch
+phrase from some popular video games.
 
-## SWBAT 2
+## Define a Local Variable With the Expected Value
+
+Open up `lib/catch_phrases.rb`. You should see the following method:
+
+```ruby
+def mario
+  status = 'Thank You Mario! But Our Princess Is In Another Castle!'
+  puts phrase
+end
+```
+
+Note that the method is trying to `puts` out a variable called `phrase`.
+
+If we run the test for this method only by typing `rspec spec/catch_phrases_spec.rb`
+into your terminal in the directory of this lab. You should see the following
+error:
+
+```ruby
+NameError:
+  undefined local variable or method `phrase' for #<RSpec::ExampleGroups::CatchPhrasesRb:0x007fa5eb399b88>
+```
+
+This error is occurring because the code inside the `#mario` method is trying to
+use the `phrase` variable **but** it's not present inside the _scope_ of the
+`#mario` method. **It is out of scope.**
+
+If we look at the test for this method in `spec/catch_phrase_spec.rb` we can see
+that it expects "It's-a me, Mario!" to be printed out.
+
+```ruby
+describe "#mario" do
+  it "puts out 'It's-a me, Mario!'" do
+    phrase = "It's-a me, Mario!"
+    expect{mario}.to output("It's-a me, Mario!\n").to_stdout
+  end
+end
+```
+
+We need to define the variable `phrase` in our `#mario` method. When `phrase` is
+called, the output should be "It's-a me, Mario!"
+
+Once `phrase` is defined in the method, the first test should pass. Let's move
+on to the next method!
+
+## Recognize the Restrictions of Local Variable Scope
```

Comment:
As above, I think this LG was no-value. This toadstool example basically fits under the previous LG.